NETWORK/getPhotofromFlicker.playground/playground.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not sure if these are required for this particular snippet. If you think so let's keep them if not please remove them. It's up to you!

Reply:
Xcode generates that piece of code automatically for the playground file.